plugins/common/functions

```diff
@@ -330,3 +330,15 @@ docker_cleanup() {
   # shellcheck disable=SC2046
   docker rmi $(docker images -f 'dangling=true' -q) &> /dev/null &
 }
+
+get_open_port() {
+  while true; do
+    local port=$(shuf -i 1025-65535 -n 1)
+    if ! nc -z 0.0.0.0 $port; then
```

Comment:
I think your if statement is missing the brackets to evaluate this and you have a continue in an iff statement?